src/app/create/Basicauction.css.ts

```diff
@@ -85,9 +89,9 @@ export const info = style({
 });
 
 export const price = style({
-  display: 'grid',
-  gridTemplateColumns: 'repeat(auto-fit,minmax(66px, auto))',
-  gap: '30px',
+  display: 'flex',
+  flexWrap: 'wrap',
+  gap: '80px',
```

Comment:
```suggestion
  display: 'grid',
  gridTemplateColumns: 'repeat(auto-fit,minmax(110px, 360px))',
  gap: '30px',
```
gap: 80px로 되어 있어서 화면을 줄이면 기존 gap: 30px과는 다른 것 같아서 grid는 어떤가유..?

Reply:
grid 기피증이 있어서..ㅠㅠ 덕분에 좋은 정보 알아갑니다!! 감사합니다~